students/14th/team3/westagram_folder/user/views.py

```diff
@@ -0,0 +1,67 @@
+import json
+import bcrypt
+import jwt
+import re
+from django.views                import View
+from django.http                 import JsonResponse, HttpResponse
+from project_westagram.settings  import SECRET_KEY
+from .models                     import Account
+
+# 치킨먹고싶다...
+class SignUpView(View):
+    def post(self, request): #포스트로 받을 시 저장
+        data = json.loads(request.body)
+        signupdb = Account.objects.all() #유저 테이블정보 다 가져와
+        try:        
+            user = Account(
+                name         = data['name'],
+                phone_number = data['phone_number'],
+                email        = data['email'],
+                password     = data['password'])
```

Comment:
코드 간격 굳!! 깔끔!
```suggestion
            user = Account(
                name         = data['name'],
                phone_number = data['phone_number'],
                email        = data['email'],
                password     = data['password']
         )
```
괄호는 아래로 내리셔서 어디까지가 객체를 선언하는 부분인지 명시해주시는 것이 좋습니다 :)

Reply:
200okay...  근데 data['password']에서 엔터 치니까 user 바로 아래로  ) 가 내려가는데, 소헌님 suggested change에는  user 보다 1 indent 앞쪽에 위치해 있는데 어느쪽이 맞나요?
![Screen Shot 2020-11-10 at 1 06 02 PM](https://user-images.githubusercontent.com/69205911/98626636-7d851c80-2355-11eb-9882-faa00401ea09.png)